test/compiler.ts

```diff
@@ -68,10 +69,10 @@ function runTests (solc, versionText) {
     }
   }
 
-  function expectError (output, errorType, message) {
+  function expectError (output: any, errorType: any, message: any) {
     if (output.errors) {
-      for (let error in output.errors) {
-        error = output.errors[error];
+      for (let errorIndex in output.errors) {
+        let error = output.errors[errorIndex];
```

Comment:
Too bad these were missed in the lint PR :)

Reply:
This was a error prior to the lint PR since this is breaking for typescript, with error being defined in the global scope multiple times.